pkgs/development/libraries/precice/default.nix

```diff
@@ -12,14 +12,21 @@ stdenv.mkDerivation rec {
   };
 
   preConfigure = ''
-    cmakeFlags="-DBUILD_SHARED_LIBS=ON -DPETSC=off"
+    cmakeFlags="-DBUILD_SHARED_LIBS=ON
+                -DPETSC=off
+                -DCMAKE_BUILD_TYPE=Debug
+                -DCMAKE_INSTALL_PREFIX=$out
+                "
   '';
-
   nativeBuildInputs = [ cmake gcc ];
   buildInputs = [ boost eigen libxml2 openmpi python2 python2Packages.numpy ];
-  installPhase = ''
-    mkdir -p $out/lib
-    cp libprecice.so libprecice.so.1.6.1 $out/lib/
+  patches = [
+    ./0001-Fix-the-install-target-dirs-to-use-the-CMAKE-flags.patch
+  ];
+  enableParallelBuilding = true;
+  postInstall = ''
+      substituteInPlace "$out"/lib/cmake/precice/preciceTargets.cmake \
+      --replace 'INTERFACE_INCLUDE_DIRECTORIES "''${_IMPORT_PREFIX}/include;' 'INTERFACE_INCLUDE_DIRECTORIES "'$out/include';'
```

Comment:
Why is this not done in the patch? Does the [other PR](https://github.com/precice/precice/pull/577) fix that?

Reply:
it dont know and its still a work in progress so I would rather keep it